### ISSUE TITLE
nvGesture training

### ISSUE THREAD

**Issue by sanchitmishra** [ahmetgunduz/Real-time-GesRec#13]
Hello Ahmet, I am trying to run your code on Nvidia dataset. On running the main.py, the train.log looks like this.
epoch	loss	acc	precision	recall	lr

1	0	0	0	0	0.1

2	0	0	0	0	0.1

3	0	0	0	0	0.1

which I don't think is right. Can you please tell me what am I doing wrong?
Other than setting path and reducing the epoch value from 100 to 50. I haven't changed anything.
[GesRec.pdf](https://github.com/ahmetgunduz/Real-time-GesRec/files/3125044/GesRec.pdf)
These are the parameters and a part of train.log after running the main.py.

**Comment by ahmetgunduz** [comment]
Please check main.py. You have the path setting wrong and the dataset is not loaded properly.. The paths are concatenated with `root`(see below).
```python
if opt.root_path != '':
        # Join some given paths with root path 
        if opt.result_path:
            opt.result_path = os.path.join(opt.root_path, opt.result_path)
        if opt.annotation_path:
            opt.annotation_path = os.path.join(opt.root_path, opt.annotation_path)
        if opt.resume_path:
            opt.resume_path = os.path.join(opt.root_path, opt.resume_path)
        if opt.pretrain_path:
            opt.pretrain_path = os.path.join(opt.root_path, opt.pretrain_path)
        if opt.video_path:
opt.video_path = os.path.join(opt.root_path, opt.video_path)```

**Comment by sanchitmishra** [comment]
Okay, I will check path again and in the dataset, I have given the path for a folder which contains video_data and other files. Is it fine or I should give exactly the path for the videos present in my dataset?

**Comment by ahmetgunduz** [comment]
Just as an example `root_path`+`video_path`+`Video_data/class_01/subject3_r0/sk_color_all` should be a proper path to the folder where you would have frames for class 01 of subject3. The last item it from one line of the `./annotation_nvGesture/trainlistall.txt`, in which every line corresponds to a different clip.

**Comment by ahmetgunduz** [comment]
@sanchitmishra  any update?

**Comment by sanchitmishra** [comment]
Hi Ahmet
I am sorry for not updating sooner, I've been busy with college stuff.
I am still facing the same issue. I have checked all the path. Also, I have tried running nv.py individually to see dataset I am getting from make_dataset() function.
the dataset looks like this.
![image](https://user-images.githubusercontent.com/14020744/57204816-b1041f80-6fd7-11e9-84ad-97abf646e66e.png)

This is my opts right now. I haven't changed anything other than epoch and setting path.
![image](https://user-images.githubusercontent.com/14020744/57204936-8e263b00-6fd8-11e9-8dbc-fa8eca0b4006.png)

And when I run the main.py,
![image](https://user-images.githubusercontent.com/14020744/57204961-b9108f00-6fd8-11e9-9b5a-c3b51110743b.png)
